hazelcast/src/main/java/com/hazelcast/spi/discovery/DiscoveryNode.java

```diff
@@ -71,5 +71,5 @@
      *
      * @return assigned properties of that node
      */
-    public abstract Map<String, Object> getProperties();
```

Comment:
Do you need to change the type of these as well? Are they connected to member attributes in any way?

Reply:
No, they could stay unchanged. It is changed as a result of chain refactoring,